integrationtest/mapper/orm/src/test/java/org/hibernate/search/integrationtest/mapper/orm/automaticindexing/association/bytype/AbstractAutomaticIndexingAssociationBaseIT.java

```diff
@@ -1153,6 +1153,8 @@ void directImplicitAssociationUpdateThroughDeleteWithAlreadyLoadedAssociation_in
 			// and the deletion even will simply be ignored.
 			containedAssociation.get( containedEntity );
 
+			containedAssociation.set( containedEntity, null );
+			containingAssociation.set( entity1, null );
```

Comment:
If you look at the comment above, this specific test only makes sense if you don't update associations.

Don't we have a test that does what you're doing here already?